interface/main/tabs/edit_roles.php

```diff
@@ -59,35 +59,82 @@
 <!DOCTYPE HTML>
 <html>
 <head>
-<?php call_required_libraries(array("jquery-min-3-1-1","bootstrap","fancybox-custom"));
+<?php call_required_libraries(array("jquery-min-3-1-1","bootstrap","font-awesome","iziModalToast"));
       resolveFancyboxCompatibility();
 ?>
     <title><?php echo xlt("Role Management") ?></title>
     <link href="js/jsoneditor/jsoneditor.css" rel="stylesheet" type="text/css">
     <script src="js/jsoneditor/jsoneditor.js"></script>
     <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/js/common.js"></script>
     <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/js/jquery-ui.js"></script>
-    <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/js/jquery.easydrag.handler.beta2.js"></script>

```

Comment:
Did you intend to remove easydrag?  This is just applicable to the fancybox, right?